help_to_heat/frontdoor/views.py

```diff
@@ -763,6 +763,9 @@ def show_question(self, session_data, question):
             epc_rating = session_data.get("epc_rating", "Not found")
             accept_suggested_epc = session_data.get("accept_suggested_epc")
             return epc_rating != "Not found" and accept_suggested_epc == "Yes"
+        if question in ["loft_access", "loft_insulation"]:
+            loft_answer = self.get_answer(session_data, "loft")
+            return loft_answer != "No, I do not have a loft or my loft has been converted into a room"
```

Comment:
seems a bit safer (and a bit more expressive) in this binary case to write as show these questions if loft answer is yes